fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/A380_842/A380Payload.tsx

```diff
@@ -79,6 +80,8 @@ export const A380Payload: React.FC<A380Props> = ({
     const [upperMidBDesired, setUpperMidBDesired] = useSeatFlags(`L:${Loadsheet.seatMap[12].simVar}_DESIRED`, Loadsheet.seatMap[12].capacity, 557);
     const [upperAftDesired, setUpperAftDesired] = useSeatFlags(`L:${Loadsheet.seatMap[13].simVar}_DESIRED`, Loadsheet.seatMap[13].capacity, 571);
 
+    const globalSettingsRedux = useSelector((state:any) => state.globalSettings);
```

Comment:
Again, my suggestion to call `useAppSelector()` instead holds true here as well

Reply:
Will be resolved in my next commit